### ISSUE TITLE
Where should we meet?

### ISSUE THREAD

**Issue by Albadylic** [fac-17/student-project-ideas#7]
**Name**
Albadylic

**Describe the problem you'd like to solve**
London is a big place and takes a long time to navigate. Finding a place to meet a friend who lives far across town can be troublesome. This app would look at the distance between each person and calculate somewhere in the middle where it would be desirable to meet. These locations could be sorted by price, what kind of restaurants there are there, what activities there are there. 

**Who are the stakeholders?**
Who would this benefit? Who would be affected by it?
Londoners, friends, relationships.

**What is currently being done to solve this problem?**
Whether it's effective or not, what is in place at the moment to solve this problem?
TFL app exists and so do many sites like TimeOut and Trip Advisor to help you find nice places to visit. Consolidating this information into an app would be sleek and helpful

**Comment by jackbridger** [comment]
cool idea

**Comment by samjam48** [comment]
also great when there are more than two people. Could factor in diet requirements as well. So even if you forget whether your mate is cealiac or dairy-free or vegan but not really vegan because they need to eat fish and eggs once a week but you don't know if they've done that yet this week
_(or perhaps the wind changed and they would be able to eat cheese today because they have been getting low on calcium... #me haha 🤣)._
Thus, you don't waste time suggesting things that don't work.

Perhaps could also factor in times like when someone gets of work and can start a journey earlier. or cost and so if one person has to pay more the other person is obliged to buy the first drink.

**Capitalist (or charitable) Stretch Goal ???**
Allow restaurants to give offers of free drinks and such to entice people to choose them as a meeting place.
e.g. https://drinki.com/ - free drinks in london app.

Show events and let people buy tickets through a partner.

Basically, make money from it. This could be charitable such as [Ecosia's new travel booking app](https://hotels.ecosia.org/?languageCode=en&currencyCode=GBP) where any hotel booking you make gives them ad money which they use to plant trees

**Comment by samjam48** [comment]
Also cool apps for showing travel times. so if you have 15 minutes it shows you how far you can get and little spots that might be much further than you think.
But it makes the point that two friends to meet at equal effort and spend might actually meet geographically much closer to the one to have the best solution rather than just in the middle.
https://www.mapnificent.net/london/#12/51.5005/-0.1283/900/51.5005/-0.1283

_(Also this is a nice map that just shows the london tube map but arranged by timings of how far to get between each stop which is cool.)_
http://london-tubemap.com/journey_times.php
